### ISSUE TITLE
Fatal Error - Not sure what causes it.

### ISSUE THREAD

**Issue by MysfitLeaf** [DevilXD/TwitchDropsMiner#650]
14:01:13: Fatal error encountered:
14:01:13: 
14:01:13: Traceback (most recent call last):
14:01:13:   File "main.py", line 155, in main
14:01:13:   File "twitch.py", line 587, in run
14:01:13:   File "twitch.py", line 747, in _run
14:01:13:   File "twitch.py", line 1660, in bulk_check_online
14:01:13:   File "asyncio\tasks.py", line 571, in _wait_for_one
14:01:13:   File "twitch.py", line 1390, in gql_request
14:01:13: exceptions.GQLException: [{'message': 'service error', 'path': ['channel', 'viewerDropCampaigns']}]
14:01:13: 
14:01:13: Exiting...
14:01:13: 
14:01:13: Application Terminated.
14:01:13: Close the window to exit the application.

Just started using this application, was loving it until this morning when I keep getting this crash. Even after restarting, still reoccurs.

**Comment by iv8512** [comment]
I also started getting a fatal error today.

```python
16:13:38: Fatal error encountered:
16:13:38: 
16:13:38: Traceback (most recent call last):
16:13:38:   File "C:\Users\Main\Documents\GitHub\TwitchDropsMiner\main.py", line 155, in main
16:13:38:     await client.run()
16:13:38:   File "C:\Users\Main\Documents\GitHub\TwitchDropsMiner\twitch.py", line 587, in run
16:13:38:     await self._run()
16:13:38:   File "C:\Users\Main\Documents\GitHub\TwitchDropsMiner\twitch.py", line 747, in _run
16:13:38:     await self.bulk_check_online(acl_channels)
16:13:38:   File "C:\Users\Main\Documents\GitHub\TwitchDropsMiner\twitch.py", line 1660, in bulk_check_online
16:13:38:     response_list = await coro
16:13:38:                     ^^^^^^^^^^
16:13:38:   File "C:\Users\Main\AppData\Local\Programs\Python\Python312\Lib\asyncio\tasks.py", line 621, in _wait_for_one
16:13:38:     return f.result()  # May raise f.exception().
16:13:38:            ^^^^^^^^^^
16:13:38:   File "C:\Users\Main\Documents\GitHub\TwitchDropsMiner\twitch.py", line 1390, in gql_request
16:13:38:     raise GQLException(response_json['errors'])
16:13:38: exceptions.GQLException: [{'message': 'service error', 'path': ['channel', 'viewerDropCampaigns']}]
16:13:38: 
16:13:38: Exiting...
16:13:39: 
16:13:39: Application Terminated.
16:13:39: Close the window to exit the application.
```

**Comment by LuWoody** [comment]
got the same error, seems to be the same issue / behavior as in #415 

effected drop this time is the "Winter Drops day 1 pt.1", game "Escape from Tarkov" which is available from "Mon, Dec 30, 2:00 PM - Tue, Dec 31, 1:59 AM GMT+1"

possible workaround is to temporarily add "Escape from Tarkov" to the exclude list

![2024-12-30_15-58-45](https://github.com/user-attachments/assets/31227ce1-0e1f-4fb5-8c4e-733d1d6450ba)

or configure it manually in the settings.json

    "exclude": {
        "__type": "set",
        "data": [
            "Escape from Tarkov"
        ]
    }

**Comment by MysfitLeaf** [comment]
Ye setting it to exclude isn't what I want, Want Tarkov drops one to work if anything.

**Comment by DevilXD** [comment]
Redirecting this to #651 since it has more discussion on this topic.

**Comment by DonatPlayZ** [comment]
Found a fix in #651 from @villario
